### PR TITLE
Rearrange folders 

### LINKED ISSUE #237: Rearrange files in the repo
During the nav reordering, we relocated the articles by changing their front matter variables, instead of moving files to different directories. The reason behind it was that `git mv` dropped the article history.

There are several ways to fix this issue and tidy up the repo. Here are some:

https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history

### PR DESCRIPTION
**Pull request by tovmasharrison**
# Description

I rearranged the folders to match the website's navbar.

Fixes #237 

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

File: concepts/crawling.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Crawling
 description: Crawling the web for machine translation training data
 ---
 
-**Crawling** or **web scraping** is the process of automatically collecting large amounts of [training data](/customisation/training-data.md) for [language models](/concepts/language-model.md) and machine translation systems.
+**Crawling** or **web scraping** is the process of automatically collecting large amounts of [training data](../features/customisation/training-data.md) for [language models](/concepts/language-model.md) and machine translation systems.
```
Comment (cefoo) [inline concepts/crawling.md:7]: The link to `training-data.md` is not working. When you click on it, it takes you to `features.md`.

**Review by cefoo — COMMENTED** [review]
Hi @tovmasharrison! Thank you for your PR!
I've reviewed links and found that there is an issue with links that lead to articles within `features.md`. Probably has something to do with `customisation.md` vs. `features.md`. Could you please check? Thanks!!

File: applications/workflows/hybrid-translation.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ They are marked as translated or approved, and potentially even locked.
 
 ### Technology
 
-The key technology for a hybrid translation workflow is translation [**quality prediction**](/quality/quality-estimation.md), which is known as *machine translation quality estimation* in the [research](/building-and-research/building-and-research.md) world.
+The key technology for a hybrid translation workflow is translation [**quality prediction**](../../quality-estimation/quality-estimation.md), which is known as *machine translation quality estimation* in the [research](../../building-and-research/building-and-research.md) world.
```
[the remark this answers is not in the document]
Comment (bittlingmayer) [reply applications/workflows/hybrid-translation.md:34]: I think we should basically never use "..", let's just use absolute paths.

**Comment by tovmasharrison** [comment]
Hi @cefoo!

I have made some changes. Can you please check and let me know if there's anything else that needs to be done? Thanks!

File: applications/commerce-and-marketplaces.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Machine translation for **commerce and marketplaces** is the translation of prod
 ## Goals
 
 * sales conversion
-* [SEO](seo.md) and site search
+* [SEO](/seo.md) and site search
```
Comment (bittlingmayer) [inline applications/commerce-and-marketplaces.md:13]: no .md

**Pull request merged**